cmd/snap/complete.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"strings"
+
+	"github.com/jessevdk/go-flags"
+
+	"github.com/snapcore/snapd/asserts"
+	"github.com/snapcore/snapd/client"
+)
+
+type installedSnapName string
+
+func (s installedSnapName) Complete(match string) []flags.Completion {
```

Comment:
It appears we currently have no tests for the new completer's. I wonder how hard it would be to write a new `complete_test.go` that mimics what we do for `TestSectionCompletion` in snap_find_test.go (and related if it would make sense to move the find completer into the complete.go code).